pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdResourceGroups.java

```diff
@@ -55,23 +55,23 @@ void run() throws PulsarAdminException {
         @Parameter(names = { "--msg-publish-rate",
                 "-mp" }, description = "message-publish-rate "
                 + "(default -1 will be overwrite if not passed)", required = false)
```

Comment:
> (default -1 will be overwrite if not passed)

Do we need to update this description ?

Reply:
[Create](https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdResourceGroups.java#L51) command not need update but [Update](https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdResourceGroups.java#L90) command need.